File: app/src/main/java/ru/mrlargha/stemobile/tools/DateFormatter.java
package ru.mrlargha.stemobile.tools;

import android.annotation.SuppressLint;

import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;

public class DateFormatter {
    public static String dateToString(Date date) {
        @SuppressLint("SimpleDateFormat") SimpleDateFormat dateFormat =
                new SimpleDateFormat("dd.MM.yy");
        return dateFormat.format(date);
    }

    public static Date stringToDate(String date) throws ParseException {
        @SuppressLint("SimpleDateFormat") SimpleDateFormat dateFormat =
                new SimpleDateFormat("dd.MM.yy");
        return dateFormat.parse(date);
    }
}

File: app/src/main/java/ru/mrlargha/stemobile/tools/DateFormatter.kt
package ru.mrlargha.stemobile.tools

import android.annotation.SuppressLint
import java.text.ParseException
import java.text.SimpleDateFormat
import java.util.*

object DateFormatter {
    fun dateToString(date: Date): String {
        @SuppressLint("SimpleDateFormat") val dateFormat = SimpleDateFormat("dd.MM.yy")
        return dateFormat.format(date)
    }

    @Throws(ParseException::class)
    fun stringToDate(date: String): Date? {
        @SuppressLint("SimpleDateFormat") val dateFormat = SimpleDateFormat("dd.MM.yy")
        return dateFormat.parse(date)
    }
}